worlds/stardew_valley/logic.py

```diff
@@ -938,18 +1253,44 @@ def has_galaxy_weapon(self) -> StardewRule:
     def has_year_two(self) -> StardewRule:
         return self.has_lived_months(4)
 
+    def has_year_three(self) -> StardewRule:
+        return self.has_lived_months(8)
+
     def has_spring_summer_or_fall(self) -> StardewRule:
```

Comment:
This function can be removed after truffle is changed to use `has_any_season_not_winter`